service/controller/resource/monitoring/scrapeconfigs/resource_test.go

```diff
@@ -148,6 +157,74 @@ func TestKVMScrapeconfigs(t *testing.T) {
 }
 
 func TestOpenStackScrapeconfigs(t *testing.T) {
+	var err error
+
+	var apps = []runtime.Object{
+		&appsv1alpha1.App{
+			ObjectMeta: v1.ObjectMeta{
+				Name:      "alice-default-apps",
+				Namespace: "org-my-organization",
+			},
+			Status: appsv1alpha1.AppStatus{
+				Version: "0.7.0",
+			},
+		},
+		&appsv1alpha1.App{
+			ObjectMeta: v1.ObjectMeta{
+				Name:      "foo-default-apps",
+				Namespace: "org-my-organization",
+			},
+			Status: appsv1alpha1.AppStatus{
+				Version: "0.7.0",
+			},
+		},
+		&appsv1alpha1.App{
+			ObjectMeta: v1.ObjectMeta{
+				Name:      "bar-default-apps",
+				Namespace: "org-my-organization",
+			},
+			Status: appsv1alpha1.AppStatus{
+				Version: "0.7.0",
+			},
+		},
+		&appsv1alpha1.App{
+			ObjectMeta: v1.ObjectMeta{
+				Name:      "baz-default-apps",
+				Namespace: "org-my-organization",
+			},
+			Status: appsv1alpha1.AppStatus{
+				Version: "0.8.0",
+			},
+		},
+		&appsv1alpha1.App{
```

Comment:
This one is running the agent also, see tests at the end of the pr

Reply:
Done